src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePicker.cs

```diff
@@ -7,4 +7,6 @@ namespace Microsoft.UI.Xaml.Controls;
 [ContentProperty(Name = nameof(Header))]
 public partial class TimePicker : Control
 {
+	internal const long DEFAULT_TIME_TICKS = -1;
```

Comment:
Please put this in `TimePicker.native.cs` instead, as the constant is not needed for non-Android and non-iOS targets